finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,150 @@
+   ```
+       
+   The following example response returns all the transactions from 1 January, 2020 to 7 June, 2021.
+
+    ```json
+    {
+        "data": [
+            {
+                "timestamp": "2021-03-04T04:00:07.285+0000",
+                "amount": -682,
+                "originatorTransactionType": "PAYMENT_FEE",
+                "originatingTransactionUuid": "77db5940-7c1d-11eb-b812-d3f21f3c0d77"
+            },
+            {
+                "timestamp": "2021-03-04T04:00:07.281+0000",
+                "amount": 3300,
+                "originatorTransactionType": "PAYMENT",
+                "originatingTransactionUuid": "77db5940-7c1d-11eb-b812-d3f21f3c0d77"
+            },
+            {
+                "timestamp": "2020-11-21T04:00:15.704+0000",
+                "amount": -621,
+                "originatorTransactionType": "CARD_PAYMENT_FEE",
+                "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+            },
+            {
+                "timestamp": "2020-11-21T04:00:15.697+0000",
+                "amount": 1100,
+                "originatorTransactionType": "CARD_PAYMENT",
+                "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+            },
+            {
+                "timestamp": "2020-09-10T09:51:35.162+0000",
+                "amount": 5925,
+                "originatorTransactionType": "FAILED_PAYOUT",
+                "originatingTransactionUuid": "d8550d7a-f347-11ea-9612-3bce5300b9a9"
+            },
+            {
+                "timestamp": "2020-09-10T09:27:28.590+0000",
+                "amount": -5925,
+                "originatorTransactionType": "PAYOUT",
+                "originatingTransactionUuid": "d8550d7a-f347-11ea-9612-3bce5300b9a9"
+            },
+            ...
+        ]
+    }
+    ```
+
+## Step 2: Fetch transactions of certain types during a specific period
+To fetch transactions of certain types from a merchant's Zettle account during a specific period, you need to fetch them from their Zettle liquid account.
+
+For example, you can fetch all card transactions.
+
+1. Fetch transactions of certain types from the merchant's Zettle liquid account. In the request path, specify transaction types as you need. See [supported transaction types](../api-reference.md#supported-transaction-types).
+   > **Tip:** You specify the organisation UUID as `self`.
+        
+   ```
+   GET /organizations/{organizationUuid}/accounts/liquid/transactions?{start}&{end}&{includeTransactionType}
+   ```
+
+   Example:
+   
+   The following example request fetches all card payments and the associated card payment fees from the merchant's Zettle liquid account from 1 January, 2020 to 7 June, 2021.
+   
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2021-06-07&includeTransactionType=CARD_PAYMENT&includeTransactionType=CARD_PAYMENT_FEE
```

Comment:
I think this example is bad, we don't want partners to fetch transactions for 1,5 year since this kind of requests can time out. 
Of course they can try if they need to. But usually an accounting period is 1 year.